src/libs/elektra/kdb.c

```diff
@@ -1595,7 +1595,7 @@ static bool runGetPhase (KeySet * backends, Key * parentKey, uint16_t phase)
  * @pre The @p handle must be a valid KDB handle as returned from kdbOpen().
  * @pre The KeySet @p returned must be a valid KeySet, i.e., constructed with ksNew().
  * @pre The KeySet @p returned must contain keys only from the `spec:/`, `dir:/`, `user:/`, `system:/`, `default:/` or `proc:/` namespaces.
- * @pre The @p parentKey Key must be a valid Key, i.e., constructed with keyNew().
+ * @pre The Key @p parentKey must be a valid Key, i.e., constructed with keyNew().
  * @pre The @p parentKey Key must not have read-only name, value or metadata.
```

Comment:
Not done for following lines.

Reply:
You only posted a suggestion for one line. I just accepted it.

Reply:
This I saw. There was text above it.

Reply:
Sorry, wasn't meant to put blame on you. It was just an explanation of what happened. I should've checked if there are similar issues nearby